corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -281,6 +281,27 @@
       <_ManagedAssembliesToLink Include="@(IntermediateAssembly)" />
       <_ManagedAssembliesToLink Include="@(_ManagedResolvedAssembliesToPublish)" />
     </ItemGroup>
+
+    <!-- Portable publish: need to supply the linker with any needed
+         reference assemblies as well.
+
+         We want to exclude assemblies that are going to be published
```

Comment:
Did you mean "assemblies that are not going to be published" ?

Reply:
I meant implementation assemblies that are going to be published, and also exist as a reference assembly. That sentence was unclear - I'll just remove it entirely.